SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Graylog Security Policy
+
+## Supported Versions
+
+Graylog is addressing vulnerabilities in the product for the current and the last releases (a release is anything that increases either the major or the minor version part, in a [semver](https://semver.org) understanding) of the last twelve months.
```

Comment:
just a minor grammatical change: we should say *for the current and the previous releases*.